recipes/mdbenchmark/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "mdbenchmark" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "4b6c64fbf6826fc0b6ecc7f5d55c5a828cf26cf700c5d51d3942a7b7d2f199af" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy >=1.8
+    - mdsynthesis
+    - click
+    - jinja2
+    - pandas
+    - matplotlib
+    - xdg <2
+    - python-levenshtein
+
+test:
```

Comment:
Test the command line tool:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands